### DISCUSSION TITLE
History timeline scroll window

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#13551] [Ideas]
While reviewing "Timeline" for a camera the red time indicator scrolls up and eventually becomes invisible as it goes off the top. I suggest to keep it centered on the panel by scrolling the timeline instead of the indicator.
Im not sure I explained this so it can be understood.
![Frig](https://github.com/user-attachments/assets/fbc2ba4d-b6b0-4879-8750-eb90fc3c7a6d)

**Comment by NickM-27** [comment]
keeping it centered does not work well, because then scrolling the timeline will always change the cameras time. Meanwhile right now you can scroll the timeline to look at the data for a certain time without affecting where you currently are. This is working as intended.

**Reply by 18923** [reply]
No problem.
2 of my cameras do not have timestamps on the image so I have to chase the timeline to see the time something occurred.

**Reply by NickM-27** [reply]
can you elaborate what you mean by chase?

**Reply by 18923** [reply]
Without a timestamp on the image the only way I know when the motion occured is by looking at the "red timestamp indicator".  What I meant by "chase" is that I constantly have to keep scrolling down on the timeline to catch up to the time indicator to keep it visible.

**Reply by NickM-27** [reply]
hmm, I believe it should automatically scroll the timeline while playback is happening. @hawkeye217 can confirm

**Reply by hawkeye217** [reply]
This has been improved in https://github.com/blakeblackshear/frigate/pull/13556
